### PR TITLE
Configure options for compatibility warning logs

### PR DESCRIPTION
**Pull request by arcanistzed**
image

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "allowBugReporter": true
   },
   "version": "1.13.0",
-  "minimumCoreVersion": "9",
```
Comment (akrigline) [inline module.json:22]: Is it practical to implement this feature on such a way that it does not initialize in v9 and this we do not need to bump this?

Comment (arcanistzed) [reply module.json:22]: Sure. It just requires a bit of extra code

**Review by akrigline — COMMENTED** [review]
Do you think it's worth it to have a boolean setting to blanket enable/disable this functionality's overrides, similar to how the `setDebugOverrides` functions?

**Comment by akrigline** [comment]
This looks great, many thanks for wiring this up!
If you think it's worth it to set up a boolean governing this functionality, go ahead and set that up, otherwise I'll merge this in and get a release in a day if I don't hear from you.

**Pull request merged**